docs/installation.md

```diff
@@ -28,6 +28,32 @@ Generate all the mocks for your project:
 
 	docker run -v "$PWD":/src -w /src vektra/mockery --all
 
+### pre-commit
+
+Add the following lines to your `.pre-commit-config.yaml` file
+
+```yaml
+repos:
+  - repo: https://github.com/ccoVeille/mockery
+    rev: v2
```

Comment:
nit/nb: It would be great if this could be automatically updated on new releases.

```suggestion
    rev: v2.43.2
```

Reply:
There is no need usually. precommit manage this well with this kind of tag